examples/convert_ieeg_to_bids.py

```diff
@@ -0,0 +1,208 @@
+"""
+====================================
+07. Convert iEEG data to BIDS format
+====================================
+
+In this example, we use MNE-BIDS to create a BIDS-compatible directory of iEEG
+data. Specifically, we will follow these steps:
+
+1. Download some iEEG data from the
+   `PhysioBank database <https://physionet.org/physiobank/database>`_.
+   and the
+   `MNE-ECoG ex <https://mne.tools/stable/auto_tutorials/misc/plot_ecog>`_.
+
+2. Load the data, extract information, and save in a new BIDS directory
+
+3. Check the result and compare it with the standard
+
+4. Confirm that written iEEG coordinates are the same before
+:func:`write_raw_bids` was called
```

Comment:
this is not yet rendered correctly

Reply:
see: https://1661-89170358-gh.circle-artifacts.com/0/html/auto_examples/convert_ieeg_to_bids.html#sphx-glr-auto-examples-convert-ieeg-to-bids-py